src/codeu/chat/client/commandline/Chat.java

```diff
@@ -40,6 +43,44 @@ public Chat(Context context) {
 		this.panels.push(createRootPanel(context));
 	}
 
+	// RECREATE 
+	//
+	// Recreates the server's last known state to display for the client 
```

Comment:
Is it just for display? Or does the memory stick around for when you fetch messages and whatnot?

In other words, if I add a user, then exit the program, and run the program again... do I have to add the user again, or is it saved from the last time?